codec-redis/src/main/java/io/netty/handler/codec/redis/RedisArrayAggregator.java

```diff
@@ -21,14 +21,13 @@
 import io.netty.util.ReferenceCountUtil;
 import io.netty.util.internal.UnstableApi;
 
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Deque;
-import java.util.List;
+import java.util.*;
 
 /**
- * Aggregates {@link RedisMessage} parts into {@link ArrayRedisMessage}. This decoder
- * should be used together with {@link RedisDecoder}.
+ * Aggregates {@link RedisMessage} parts into {@link ArrayRedisMessage} or {@link SetRedisMessage}.
+ * This decoder should be used together with {@link RedisDecoder}.
+ *
+ * todo rename RedisArrayAggregator to RedisCollectionAggregator?
```

Comment:
cleanup ?

Reply:
Due to introduce set types, do we need to rename `RedisArrayAggregator` to `RedisCollectionAggregator`?

`RedisCollectionAggregator` no only aggregate Array types message, but also aggreate Set types messags.

Reply:
I had cleanup this